worlds/sm64ex/docs/en_Super Mario 64 EX Advance Setup.md

```diff
@@ -0,0 +1,71 @@
+# Super Mario 64 EX Advanced Setup Guide
+
+## Required Software
+
+- Super Mario 64 US Rom (Japanese may work also. Europe and Shindou not supported)

```

Comment:
Not `may`. Japanese is also fully supported.

Reply:
fixed